libs/langchain/langchain/smith/evaluation/runner_utils.py

```diff
@@ -988,21 +1008,26 @@ def _collect_test_results(
                 )
             elif isinstance(callback, LangChainTracer):
                 run = callback.latest_run
+                example_id = callback.example_id
                 execution_time = (
                     (run.end_time - run.start_time).total_seconds()
                     if run and run.end_time
                     else None
                 )
+                all_execution_time[str(example_id)] = execution_time
 
-    results = {}
+    results: dict = {}
     for example, output in zip(examples, batch_results):
         feedback = all_eval_results.get(str(example.id), [])
         results[str(example.id)] = {
-            "output": output,
             "input": example.inputs,
             "feedback": feedback,
-            "execution_time": execution_time,
+            "execution_time": all_execution_time.get(str(example.id)),
         }
+        if isinstance(output, EvalError):
+            results[str(example.id)]["error"] = output.error
```

Comment:
Was this `Error` previously? Could this end up being a breaking change?

Reply:
hm ya will update